src/main/webapp/app/entities/event-attendee/event-attendee-row.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Button } from 'reactstrap';
+import { IEventAttendee } from 'app/shared/model/event-attendee.model';
+import './event-attendee.scss';
+
+export interface IEventAttendeeTableRowProps {
+  user: IEventAttendee;
+  index: number;
+}
+
+export class EventAttendeeRow extends React.Component<IEventAttendeeTableRowProps> {
+  state = { user: this.props.user };
+
+  contactUser = () => {
+    window.open(`https://wa.me/${this.props.user.contactNumber}`, '_blank');
+  };
+
+  render() {
+    const { user, index } = this.props;
+    return (
+      <tr key={`event-attendee-${index + 1}`}>
+        <td scope="row">{index + 1}</td>
+        <td>{user.userName}</td>
+        <td>{user.year}</td>
+        <td>{'provideTransport' in user ? <FontAwesomeIcon icon="car" /> : null}</td>
```

Comment:
`'provideTransport' in user`
maybe `user.provideTransport` is better?

Reply:
Resolve by https://github.com/Five-Fishes/Club-Management-React-Client/pull/57/commits/91ad3f60f40383ef447cd55a57334d19e731546d